webpack.config.dev.js

```diff
@@ -4,11 +4,11 @@ const webpack = require('webpack');
 const exec = require('child_process').exec;
 const WebpackWatchPlugin = require('webpack-watch-files-plugin').default;
 
-const env = 'development';
+const env = process.env.APP_ENV || 'development';
 assert(['local', 'cloud-gov', 'development', 'staging', 'uat', 'production', 'ddev'].includes(env), `${env} is not an acceptable environment.`);
 
 module.exports = {
-  mode: env,
+  mode: 'development',
```

Comment:
@mrclay I was expecting this to stay as "env". Why hardcode "development" here, given the other changes you made?

Reply:
webpack mode can only be production or development, and this file used to hardcode it to development, so I figured development mode was still desired.